src/commands/Checkin.ts

```diff
@@ -40,7 +40,7 @@ export default class Checkin extends Command {
         boardRequired: true,
         enabled: true,
         description:
-          "Sends a private message with all check-in codes from today's events. Calling with `now` argument sends public embed of checkin code if any events are now live!",
+          "Sends a private message with all check-in codes from today's events. Calling with `public` argument sends public embed of checkin code.",
```

Comment:
Maybe here it should specify that public sends in the current channel instead of via DM?